src/index.test.js

```diff
@@ -129,6 +129,21 @@ const tests = {
       {name: {first: 'bat'}},
     ],
   },
+  'can handle property callback': {
+    input: [
+      [
+        {name: {first: 'baz'}},
+        {name: {first: 'bat'}},
+        {name: {first: 'foo'}},
+      ],
+      'ba',
+      {keys: [(item) => item.name.first]},
```

Comment:
This is the reason the build is failing. That arrow function doesn't need parentheses.